sfm_pc/management/commands/download_data.py

```diff
@@ -0,0 +1,222 @@
+        )
+
+        parser.add_argument(
+            '--entity_doc_id',
+            dest='entity_doc_id',
+            default='1o-K13Od1pGc7FOQ-JS8LJW9Angk3_UPZHTmOU7hD3wU',
+            help='Import data from specified Google Sheets Document'
+        )
+        
+        parser.add_argument(
+            '--country_code',
+            dest='country_code',
+            help='Country code for the import'
+        )
+
+    def handle(self, *args, **kwargs):
+        entity_doc_id = kwargs['entity_doc_id']
+        location_doc_id = kwargs['location_doc_id']
+        sources_doc_id = kwargs['sources_doc_id']
+        country_code = kwargs['country_code'].rstrip()
+        
+        parent_directory = f'data'
+        country_directory = f'{parent_directory}/countries/{country_code}'
+
+        self._create_csv_files(
+            entity_doc_id=entity_doc_id,
+            sources_doc_id=sources_doc_id,
+            country_directory=country_directory,
+            sources_directory=parent_directory
+        )
+
+        self._create_location_file(location_doc_id, country_directory)
+    
+    def _create_location_file(self, location_doc_id, output_directory):
+        downloader, buffer = self._build_location_downloader(location_doc_id)
+        
+        self.stdout.write('Downloading locations file...')
+        
+        done = False
+
+        with tqdm(total=100) as progress_bar:
+            while not done:
+                status, done = downloader.next_chunk()
+                progress_bar.update(int(status.progress() * 100) - progress_bar.n)
+
+        buffer.seek(0)
+        
+        target = f'{output_directory}/locations.geojson'
+
+        with open(target, 'wb') as f:
+            f.write(buffer.getbuffer())
+
+        self.stdout.write(
+            self.style.SUCCESS('Downloaded locations file to {}'.format(target))
+        )
+    
+    def _build_location_downloader(self, location_doc_id):
+        drive_service = self._build_google_service(
+            scopes=['https://www.googleapis.com/auth/drive.readonly'],
+            service='drive',
+            version='v3'
+        )
+
+        request = drive_service.files().get_media(fileId=location_doc_id)
+        
+        location_buffer = io.BytesIO()
+        return MediaIoBaseDownload(location_buffer, request), location_buffer
+    
+    def _create_csv_files(
+            self,
+            entity_doc_id,
+            sources_doc_id,
+            country_directory,
+            sources_directory
+        ):
+        
+        sheets_service = self._build_google_service(
+            scopes=['https://www.googleapis.com/auth/spreadsheets.readonly'],
+            service='sheets',
+            version='v4'
+        )
+        
+        self._create_sources_file(
+            sheets_service=sheets_service,
+            document_id=sources_doc_id,
+            output_directory=sources_directory
+        )
+        
+        self._create_entity_files(
```

Comment:
`_create_sources_file` and `_create_entity_files` are doing very similar things but slightly different (we're getting the source file based on the key value). i wavered back and forth over whether or not to separate this (first version had them separate), but decided to include them together. i thought instantiating the sheets service once made sense.